prow/plugins/plugins.go

```diff
@@ -44,49 +47,49 @@ var (
 	statusEventHandlers        = map[string]StatusEventHandler{}
 )
 
-type IssueHandler func(PluginClient, github.IssueEvent) error
+type IssueHandler func(PluginClient, Config, github.IssueEvent) error
```

Comment:
I wasn't sure if it quite fit there, as the plugins would then need to resolve which config by looking at their own repo and org? That seemed like something that should happen before the registered handlers were called